src/NLog.Web/README.md

```diff
@@ -0,0 +1,30 @@
+# NLog.Web
+
+- Logger Provider for ASP.NET platform.
+- Extends logging output with details from the active HttpContext via NLog [LayoutRenderers](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web), just by changing NLog.config file.
+- Capable to log to many destinations via NLog [Targets](https://nlog-project.org/config/?tab=targets).
+- Supports logging output in standard formats like JSON, CVS, W3C ELF and XML using NLog [Layouts](https://nlog-project.org/config/?tab=layouts).
+- Supports advanced HTTP pipeline interception via provided IHttpModule classes, including logging of HTTP request body.
```

Comment:
```
- Supports HttpModule injection for [HTTP Request Logging](https://github.com/NLog/NLog.Web/wiki/HTTP-Request-Logging)
```
Think this bullet should be moved up the list. Just below the bullet with NLog LayoutRenderers